### PR TITLE
fix issue1212

### PR DESCRIPTION
**Pull request by leohou0430**
// Describe your PR here; eg. Fixes #issueNo

Make sure that:

- [ ] You have read the [contribution guidelines](https://dromara.org/projects/soul/contributor/).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: soul-admin/src/main/java/org/dromara/soul/admin/listener/websocket/WebsocketListener.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        }
+    }
+
+    @Override
+    public void requestInitialized(final ServletRequestEvent sre) {
+        try {
+            HttpServletRequest request = (HttpServletRequest) sre.getServletRequest();
+            if (null != request && null != request.getSession()) {
+                HttpSession session = request.getSession();
+                //把HttpServletRequest中的IP地址放入HttpSession中，关键字可任取，此处为ClientIP
```
[the remark this answers is not in the document]
Comment (leohou0430) [reply soul-admin/src/main/java/org/dromara/soul/admin/listener/websocket/WebsocketListener.java:63]: OK

File: soul-admin/src/main/java/org/dromara/soul/admin/listener/websocket/WebsocketConfigurator.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ * @author huangxiaofeng
+ * @since 2.0.0
+ */
+@Slf4j
+public class WebsocketConfigurator extends ServerEndpointConfig.Configurator {
+
+    @Override
+    public void modifyHandshake(final ServerEndpointConfig sec, final HandshakeRequest request, final HandshakeResponse response) {
+        HttpSession httpSession = (HttpSession) request.getHttpSession();
+        // 把HttpSession中保存的ClientIP放到ServerEndpointConfig中，关键字可以跟之前不同
```
Comment (yu199195) [inline soul-admin/src/main/java/org/dromara/soul/admin/listener/websocket/WebsocketConfigurator.java:40]: modify english

File: soul-sync-data-center/soul-sync-data-websocket/src/main/java/org/dromara/soul/plugin/sync/data/websocket/WebsocketSyncDataService.java [hunk 3]
```diff
@@ -71,7 +71,7 @@ public WebsocketSyncDataService(final WebsocketConfig websocketConfig,
         }
         try {
             for (WebSocketClient client : clients) {
-                boolean success = client.connectBlocking(3000, TimeUnit.MILLISECONDS);
+                boolean success = client.connectBlocking(3, TimeUnit.SECONDS);
```
Comment (yu199195) [inline soul-sync-data-center/soul-sync-data-websocket/src/main/java/org/dromara/soul/plugin/sync/data/websocket/WebsocketSyncDataService.java:74]: donot modify this

**Comment by yu199195** [comment]
please check ci

**Pull request merged**